docs/topics/data-analysis/data-science-libraries.md

```diff
@@ -1,88 +1,40 @@
+| [ND4J](https://github.com/eclipse/deeplearning4j/tree/master/nd4j)     | - Data cleaning and processing  <br/> - Model building                                            | - Matrix mathematics library for the JVM <br/> - Over 500 mathematical, linear algebra, and deep learning operations                                                                                                                                                                                                                                                                                        |
+| [Apache Commons Math](https://commons.apache.org/proper/commons-math/) | - Data cleaning and processing  <br/> - Model building                                            | - Mathematics and statistics operations for Java <br/> - Correlations, distributions, linear algebra, geometry, and other operations <br/> - Machine learning models                                                                                                                                                                                                                                        |
+| [NM Dev](https://nm.dev/)                                              | - Data cleaning and processing  <br/> - Model building                                            | - Java math library of numerical algorithms <br/> - Object-oriented numerical methods <br/> -  Linear algebra, optimization, statistics, calculus, and more operations                                                                                                                                                                                                                                      |
+| [Apache OpenNLP](https://opennlp.apache.org/)                          | - Data cleaning and processing  <br/> - Model building                                            | - Machine-learning-based toolkit for the processing of natural language text <br/> - Tokenization, sentence segmentation, part-of-speech tagging, and other tasks <br/> - Built-in tools for data modeling and model validation                                                                                                                                                                             |
+| [Charts](https://github.com/HanSolo/charts)                            | Data exploration and visualization                                                                | - Scientific [JavaFX](https://openjfx.io/) charting library under development                                                                                                                                                                                                                                                                                                                               |
```

Comment:
This one sticks out a little compared to the other tools (its only feature being that something is under development)

Reply:
I added more details.